extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesConfig.java

```diff
@@ -92,6 +92,8 @@
 
     /**
      * The host under which the application is going to be exposed
+     * 
+     * @deprecated Use the {@code quarkus.kubernetes.ingress.host} instead
```

Comment:
I wonder if we should try to add an automated warning when using deprecated configuration. Similar to the one we have for unused properties.

IMO, that's food for thoughts for later improvements but as for this PR is concerned, I wonder if we should at least add a warning when using these old configurations properties? Wondering also if we should handle the case when there is a conflict? I think the new ones should be prioritized but we should probably have a warning too.

Reply:
I updated the code such as, the old properties are read only if new ones are not set. and in that case, there is a warning.